Persistence.MongoDB/Serializers/TransactionTypeSerializer.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization;
+using Persistence.Models;
+
+namespace Persistence.MongoDB.Serializers
+{
+    public class TransactionTypeSerializer : EnumToStringUsingTranslationMappingSerializer<TransactionType>
+    {
+        public static readonly TransactionTypeSerializer Instance = new TransactionTypeSerializer();
+
+        private TransactionTypeSerializer() : base(new Dictionary<TransactionType, string>
```

Comment:
I'm pretty sure there's more (or there should be anyway) if not, then that's probably an issue with our current core that we need to fix

Reply:
i got this list by getting a fresh tokentransaction dump and doing a distinct(). Guess I forgot pokeyentransactions are a thing too. maybe this shouldn't be an enum after all, but a string